python/core/qgsjsonutils.sip

```diff
@@ -1,207 +1,273 @@
+.. seealso:: setAttributes()
+.. seealso:: excludedAttributes()
+.. note::
+
+   Attributes excluded via excludedAttributes() take precedence over
+ attributes returned by this method.
+ :rtype: QgsAttributeList
+%End
 
-    /** Sets a list of attributes to specifically exclude from the JSON exports. Excluded attributes
-     * take precedence over attributes included via setAttributes().
-     * @param attributes list of attribute indexes to exclude
-     * @see excludedAttributes()
-     * @see setAttributes()
-     */
     void setExcludedAttributes( const QgsAttributeList &attributes );
+%Docstring
+ Sets a list of attributes to specifically exclude from the JSON exports. Excluded attributes
+ take precedence over attributes included via setAttributes().
+ \param attributes list of attribute indexes to exclude
+.. seealso:: excludedAttributes()
+.. seealso:: setAttributes()
+%End
 
-    /** Returns a list of attributes which will be specifically excluded from the JSON exports. Excluded attributes
-     * take precedence over attributes included via attributes().
-     * @see setExcludedAttributes()
-     * @see attributes()
-     */
     QgsAttributeList excludedAttributes() const;
+%Docstring
+ Returns a list of attributes which will be specifically excluded from the JSON exports. Excluded attributes
+ take precedence over attributes included via attributes().
+.. seealso:: setExcludedAttributes()
+.. seealso:: attributes()
+ :rtype: QgsAttributeList
+%End
 
-    /** Returns a GeoJSON string representation of a feature.
-     * @param feature feature to convert
-     * @param extraProperties map of extra attributes to include in feature's properties
-     * @param id optional ID to use as GeoJSON feature's ID instead of input feature's ID. If omitted, feature's
-     * ID is used.
-     * @returns GeoJSON string
-     * @see exportFeatures()
-     */
-    QString exportFeature( const QgsFeature& feature,
-                           const QVariantMap& extraProperties = QVariantMap(),
+    QString exportFeature( const QgsFeature &feature,
+                           const QVariantMap &extraProperties = QVariantMap(),
                            const QVariant &id = QVariant() ) const;
+%Docstring
+ Returns a GeoJSON string representation of a feature.
+ \param feature feature to convert
+ \param extraProperties map of extra attributes to include in feature's properties
+ \param id optional ID to use as GeoJSON feature's ID instead of input feature's ID. If omitted, feature's
+ ID is used.
+ :return: GeoJSON string
+.. seealso:: exportFeatures()
+ :rtype: str
+%End
 
 
-    /** Returns a GeoJSON string representation of a list of features (feature collection).
-     * @param features features to convert
-     * @returns GeoJSON string
-     * @see exportFeature()
-     */
     QString exportFeatures( const QgsFeatureList &features ) const;
-
-  private:
-
-    QgsJSONExporter( const QgsJSONExporter& );
```

Comment:
I suspect in this case it was an error - the c++ class allows copies and I can't see any reason they should be blocked in python.